src/js/view/controls/settings-menu.js

```diff
@@ -54,6 +51,12 @@ export function createSettingsMenu(controlbar, onVisibility, localization) {
     return settingsMenu;
 }
 
+function showSettingsMenuIcon(settingsMenu, controlbar) {
+    // Show or hide settings menu icon dependant on amount of submenus
+    const submenuNames = settingsMenu.getSubmenuNames();
+    const toggleIcon = submenuNames.indexOf('quality') > -1 || submenuNames.indexOf('playbackRates') > -1;
```

Comment:
How about using `some` instead of `indexOf` twice?
```js
const toggleIcon = submenuNames.some(name => (name === 'quality' || name === 'playbackRates'));
```